src/ENDFtk/syntaxTree/File.hpp

```diff
@@ -52,4 +52,7 @@ class File {
   int MF() const { return this->fileNo; }
 
   int fileNumber() const { return this->MF(); }
+
+  std::vector< int > sectionNumbers() const { 
```

Comment:
Now why would you take a nice lazy view and force the client to heap allocate memory for a container?

```suggestion
  auto sectionNumbers() const { 
```